src/commands/render-image.ts

```diff
@@ -81,3 +104,63 @@ async function createFile(
     ws.on('finish', resolve);
   });
 }
+
+function generateHtml(streamKey: string): string {
+  return `<html>
+  <head>
+    <meta charset="utf-8" />
+    <link
+      rel="stylesheet"
+      href="https://unpkg.com/@vertexvis/viewer@0.9.x/dist/viewer/viewer.css"
+    />
+    <script
+      type="module"
+      src="https://unpkg.com/@vertexvis/viewer@0.9.x/dist/viewer/viewer.esm.js"
+    ></script>
+    <script
+      nomodule
+      src="https://unpkg.com/@vertexvis/viewer@0.9.x/dist/viewer.js"
+    ></script>
+  </head>
+  <body>
+    <vertex-viewer id="viewer" class="viewer" style="width: auto; height: auto">
```

Comment:
We'll be requiring a client ID in the near future. I think we should add that here as well. Note, recommendation would be to have a different client ID than what is used by internal APIs.

Reply:
So this is the actual applications client id? What is the key? `clientId="xxx"`?

Reply:
> So this is the actual applications client id?

Yep.

> What is the key?

`<vertex-viewer client-id="123">`